### PR TITLE
Rename Python package to meteofrance_api

### PR DESCRIPTION
**Pull request by oncleben31**
Old Python package name (`meteofrance`) was for retro compatibility with another Python package managing weather forecast from Météo-France.
We prefer to have same name for package and project.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,9 +9,6 @@ homepage = "https://github.com/hacf-fr/meteofrance-api"
 repository = "https://github.com/hacf-fr/meteofrance-api"
 documentation = "https://meteofrance-api.readthedocs.io"
 keywords = ["weather", "weather-api", "meteo"]
-packages = [
-    { include = "meteofrance", from = "src" },
-]
```
Comment (Quentame) [inline pyproject.toml:14]: no need of this ?

**Review by Quentame — APPROVED** [review]
Thanks :+1: 

Looks good to me.

**Comment by Quentame** [comment]
Should we release and update HA after that ?
Or maybe you have something else to work on before ?

**Comment by oncleben31** [comment]
Yes and I propose to bump package version to 1.0.0

**Pull request merged**